src/app/forms/templates/general/body.template.ts

```diff
@@ -1,39 +1,35 @@
-import { MultiOptionsService, SpecialRefData } from '@forms/services/multi-options.service';
-import getOptionsFromEnum from '@forms/utils/enum-map';
+import { ValidatorNames } from '@forms/models/validators.enum';
```

Comment:
We spoke a while ago about slowly removing any code from templates and making them pure JSON, this isn't mandatory but does make sense to do them as we go if possible

Reply:
There are still at a lot of templates that encapsulate some switching logic, but ok, sure.

`body.template.ts` is now deleted and instead broken down into:
- `psv-body.template.ts`
- `hgv-trl-body.template.ts`